splipy/utils/image.py

```diff
@@ -156,7 +156,7 @@ def image_curves(filename):
 
         # start off with a uniform(ish) knot vector
         knot = []
-        nStart = min(n/10, 90)
+        nStart = int(min(n/10, 90))
```

Comment:
The reason Py3 gives you floats here is because `/` is true division, while `//` is integer division. So I think we should use `min(n//10, 90)` instead.

Reply:
Was not aware of the `//`-operator. It is a neat trick, but you're actually not guaranteed to get an int out. 
```python
>>> print(type(10.0//2))
<class 'float'>
```
However with integer input, you do get an integer out which kind of was the old structure of the thing anyway